library/constant_time.c

```diff
@@ -81,7 +81,7 @@ unsigned mbedtls_ct_uint_mask( unsigned value )
 #endif
 }
 
-#if defined(MBEDTLS_SSL_SOME_SUITES_USE_TLS_CBC)
+#if defined(MBEDTLS_SSL_SOME_SUITES_USE_MAC)
```

Comment:
It's interesting that we came to [simillar results](https://github.com/Mbed-TLS/mbedtls/pull/6385/files#diff-63a9649f9ecb93f17d935dfdc33a6b129ae5f80e9f675313bf3869eb46e40db8R217), although mine seem to be an overkill :)